hospital/src/main/java/by/emel/anton/model/beans/therapy/Therapy.java

```diff
@@ -32,8 +44,32 @@ public LocalDate getEndDate() {
         return endDate;
     }
 
+    public Patient getPatient() {
+        return patient;
+    }
+
+    public void setId(int id) {
+        this.id = id;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    public void setStartDate(LocalDate startDate) {
+        this.startDate = startDate;
+    }
+
+    public void setEndDate(LocalDate endDate) {
+        this.endDate = endDate;
+    }
+
+    public void setPatient(Patient patient) {
+        this.patient = patient;
+    }
+
     @Override
     public String toString() {
-        return id + Constants.SEPARATOR + description + Constants.SEPARATOR + startDate + Constants.SEPARATOR + endDate;
+        return id + Constants.SEPARATOR + description + Constants.SEPARATOR + startDate + Constants.SEPARATOR + endDate + Constants.SEPARATOR + patient.getId();
```

Comment:
You can use String.join method to avoid SEPARATOR duplication

Reply:
done